.editorconfig

```diff
@@ -3,3 +3,6 @@
 [*]
 binary_next_line = true
 switch_case_indent = true
+
+[openwrt/**]
```

Comment:
Why these lines are added?

Reply:
@pandafy what do these lines mean? Do they mean to ignore the openwrt directory which is used by runbuild to compile openwrt? If so can you please add a comment?

Reply:
Yes, these lines are added to avoid formatting of scripts in openwrt directory. I have added a comment in latest comment. 
@devkapilbansal sorry I missed your ping.